src/docs/product/accounts/quotas/manage-transaction-quota.mdx

```diff
@@ -2,51 +2,157 @@
 
-## 1. SDK Configuration: Tracing Options {#2-sdk-configuration-tracing-options}
+Applying the proper SDK configuration is an iterative and on-going process, but these tips will show you how to best use Sentry's tools to get the most out of your transaction events. These tips are ordered from easiest or least time-consuming to most challenging or potentially time-consuming. The first two options are things you can do in [sentry.io](https://sentry.io), whereas the remaining ones are things you'll do in the SDK.
 
-When you configure your the SDK, you can control the number of transactions that are sent to Sentry by setting the [tracing options](/platform-redirect/?next=/configuration/options/%23tracing-options). You can also set up [custom instrumentation](/platform-redirect/?next=/performance/instrumentation/custom-instrumentation/) for performance monitoring to capture certain types of transactions.
+## Before You Begin: Check Your Quota Usage {#3-event-usage-stats}
+
+You can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer key questions about the breakdown of your incoming events or which projects are consuming your quota. The answers to these questions can help you figure out where you need to do further fine-tuning of your SDK filters and configuration.
+
+This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
+
+### How can I see a breakdown of incoming events?
+
+The [Usage Stats](/product/stats/#usage-stats) tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: _accepted_, _dropped_, or _filtered_. Only accepted events affect your quota:
+
+![Usage Stats page showing transactions](usage-stats-transactions.png)
+
+### What are my busiest projects?
+
+The "Project" table in the "Usage Stats" tab of **Stats** breaks down your events by project so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings, where you can manage its [inbound filters](#2-inbound-data-filters).
+
+### Which transactions are consuming my quota?
+
+You can set up a query in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:
```

Comment:
Is it worth setting up a warning here to indicate that this search is only possible on Business plan? You could also create this query and use org redirects so that customers could easily navigate right to the page with the query preselected like so:

https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-title&statsPeriod=14d&yAxis=count()